cves/CVE-2005-2970.yml

```diff
@@ -43,17 +53,19 @@ bounty:
   url:
 reviews: []
 bugs: []
-repo:
+repo: https://github.com/apache/httpd
 fixes_vcc_instructions: |
   Please put the commit hash in "commit" below (see my example in
   CVE-2011-3092.yml). Fixes and VCCs follow the same format.
 fixes:
+   - commit: 6a28069cef904802aeb334adce5e210a7379194c
+     note: 
```

Comment:
Some commit message or description on what was fixed here would be good

Reply:
Added general description of the fix